specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/types/privateAccesses.json

```diff
@@ -29,6 +29,11 @@
       "type": "object",
       "description": "The properties of a private access resource",
       "properties": {
+        "provisioningState": {
+          "$ref": "./common.json#/definitions/provisioningState",
+          "description": "Most recent provisioning state for the given privateAccess resource.",
+          "readOnly": true
```

Comment:
readonly should be added to the common type definition

Reply:
Do you mean by moving readOnly flag to common type or simply adding the flag to common type definition as well?